stacks/loki/README.md

```diff
@@ -93,7 +93,7 @@ The `loki-stack` has custom default Helm values. See the [values](./values.yml)
 To inspect the stack's current values, run the following command:
 
 ```console
-helm show values grafana/loki-stack --version 2.5.1
+helm show values grafana/loki-stack --version 2.6.3
 ```
 
 To change these values, open the Helm values file `values.yml`, change whatever values you want, save and exit the file, and apply the changes by running `helm upgrade` command:
```

Comment:
The old version (**2.5.1**) is still present in the **helm upgrade** command. Should be:

```console
helm upgrade loki grafana/loki-stack --version 2.6.3 \
  --namespace loki-stack \
  --values values.yml
```

Reply:
Latest commit has everything